tsstats/templates/index.jinja2

```diff
@@ -27,17 +27,14 @@
   </style>
 </head>
 <body>
-<nav class="navbar navbar-toggleable-md navbar-light bg-faded sticky-top">
-  <div class="d-flex justify-content-between hidden-lg-up">
-    <a href="" class="navbar-brand">{{ title }}</a>
-    <button class="navbar-toggler hidden-lg-up" type="button" data-toggle="collapse" data-target="main-nav" aria-controls="main-nav" aria-expanded="false" aria-label="Toggle navigation">
-      <span class="navbar-toggler-icon"></span>
-    </button>
-  </div>
+<nav class="navbar navbar-expand-md navbar-light bg-faded sticky-top">
```

Comment:
`bg-faded` seems to be [removed](https://getbootstrap.com/docs/4.0/utilities/colors/).
Please change it to `bg-light` or `bg-white`.

Reply:
Good catch!. Stuff like this is why I'm a big.fan of code review. I'll fix it up soon.